src/plugins/code-golf/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { Component, jsx } from "#DCGView";
+import { Inserter, PluginController } from "../PluginController";
+import "./index.less";
+import { format } from "localization/i18n-core";
+import { IfElse } from "src/components";
+import { ExpressionModel } from "src/globals";
+
+export class ExpressionItemCostPanel extends Component<{
+  model: () => ExpressionModel;
+  el: () => HTMLDivElement;
+}> {
+  rootblock: Element | null | undefined = null;
```

Comment:
What's the difference between what `null` represents and what `undefined` represents?

Reply:
No difference, and given what I remember about this variable, `undefined` is likely unnecessary altogether. I probably don't even need this variable (rootblock) at all anymore since I initially added it in to fix a bug that I circumvented another way later (the component wouldn't always have access to the HTML element containing the mathquill)